docs/contributing/installation_for_development.rst

```diff
@@ -13,11 +13,7 @@ To set up your development environment:
       git clone https://github.com/HinodeXRT/xrtpy.git`
       cd xrtpy
 
-2. Install the required dependencies::
-
-      pip install -r requirements.txt
-
-3. Install the package in editable mode::
+2. Install the package in editable mode::
 
       pip install -e .
```

Comment:
The `pip install -e .` command will install the requirements, so the previous step isn't necessary.